tests/Aspire.Hosting.MySql.Tests/MySqlFunctionalTests.cs

```diff
@@ -442,4 +442,79 @@ await pipeline.ExecuteAsync(async token =>
             Assert.Equal("BatMobile", cars[0].Brand);
         }, cts.Token);
     }
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    [RequiresDocker]
+    public async Task MySql_WithPersistentLifetime_ReusesContainers(bool useMultipleInstances)
```

Comment:
it may help to add a blurb about why we are testing multiple instances. It took me a second to understand why.